src/components/activity-header/account-owner.scss

```diff
@@ -0,0 +1,26 @@
+@import "../vars.scss";
+
+  .accountOwner {
```

Comment:
This camel case naming convention came from the CSS modules used in the portal dashboard, but is not the desired naming convention here.  We want to stick with kebab case in the activity player.  So in this case, we would use `account-owner`.